### ISSUE TITLE
SOE in lift.logError client side js.

### ISSUE THREAD

**Issue by hedefalk** [lift/framework#1720]
```
>window.lift.logError("hello");

lift.js:1 Uncaught RangeError: Maximum call stack size exceeded
    at Object.window.lift.logError (http://localhost:8081/classpath/lift.js:1:8238)
    at Object.lift_settings.logError (http://localhost:8081/lift/page/F276291420657F5GK33.js:2:264)
    at Object.window.lift.logError (http://localhost:8081/classpath/lift.js:1:8259)
    at Object.lift_settings.logError (http://localhost:8081/lift/page/F276291420657F5GK33.js:2:264)
    at Object.window.lift.logError (http://localhost:8081/classpath/lift.js:1:8259)
    at Object.lift_settings.logError (http://localhost:8081/lift/page/F276291420657F5GK33.js:2:264)
    at Object.window.lift.logError (http://localhost:8081/classpath/lift.js:1:8259)
    at Object.lift_settings.logError (http://localhost:8081/lift/page/F276291420657F5GK33.js:2:264)
    at Object.window.lift.logError (http://localhost:8081/classpath/lift.js:1:8259)
    at Object.lift_settings.logError (http://localhost:8081/lift/page/F276291420657F5GK33.js:2:264)

```

https://groups.google.com/forum/#!topic/liftweb/FKBlmGxMJZE

**Comment by joescii** [comment]
Easily reproduced: https://github.com/joescii/lift-SOE-1720

**Comment by joescii** [comment]
It appears we last touched this code a year ago: https://github.com/lift/framework/tree/fd9ba88be6638c2c253a43bbc69409158f0510e5 

Ay @Shadowfiend: Y U no test ur code, bruh?  And why doesn't GitHub feature a troll-face emoticon?? :-1:

**Comment by joescii** [comment]
Ok, so new Lift committer just learning out loud here.  I tried 2.6.2 to see if the bug existed then, and now I have learned that `logError` is a new thing in 3.0.  I see that [`LiftRules.jsLogFunc` used to call `liftAjax.lift_defaultLogError`](https://github.com/lift/framework/blob/2.6-release/web/webkit/src/main/scala/net/liftweb/http/LiftRules.scala#L779-L793).  It seems to me that the intention here is that the default `logError` should call the private `consoleOrAlert` function in the `lift` JS object.  Based on how we used to do this, it seems to make sense that we need a `defaultLogError` function which can do this, since `consoleOrAlert` appears to be purposefully private.
